integration_tests/src/main/python/cast_test.py

```diff
@@ -294,14 +381,37 @@ def _assert_cast_to_string_equal (data_gen, conf):
         conf
     )
 
+# split all_array_gens_for_cast_to_string
+# remove below split and merge tests: "TODO sub-issue in https://github.com/NVIDIA/spark-rapids/issues/9653 to support non-UTC tz for Cast from Date/Timestamp to String"
+gens_for_non_utc_strs = [
```

Comment:
nit: Again here I think we really only need Array(date) and Array(Timestamp).  The rest are nice, but not required.